mypy_django_plugin/transformers/manytomany.py

```diff
@@ -159,13 +159,15 @@ def get_related_manager_and_model(ctx: MethodContext) -> Optional[Tuple[Instance
         # Returning a 'ManyRelatedManager'. Which we want to, just like Django, build from the
         # default manager of the related model.
         many_related_manager = ctx.default_return_type
-        # Require first type argument of 'ManyRelatedManager' to be a model
+        # Require first and second type argument of 'ManyRelatedManager' to be models
         if (
-            many_related_manager.args
+            len(many_related_manager.args) >= 2
```

Comment:
Can this regress users who only annotated 1 typevar out of 2? Or is this function only dealing with plugin-generated ManyRelatedManager?

Reply:
No this code is only reached via one of the overloads of `ManyToManyField.__get__`, namely the one when there's a model instance and a `ManyRelatedManager` is returned.

Here's the condition:

https://github.com/typeddjango/django-stubs/blob/0a006f2dda989679b3d09fc876677bd90e1e5cb4/mypy_django_plugin/main.py#L193-L197